qa-anvil.planx-pla.net/manifest.json

```diff
@@ -7,7 +7,7 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "quay.io/cdis/arborist:1.1.2",
+    "arborist": "quay.io/cdis/arborist:master",
```

Comment:
should this one also have deployment version 2 or is this intentional?

Reply:
shoot, good catch. Missed this one